src/hlsBinaries.ts

```diff
@@ -221,7 +221,7 @@ async function getLatestReleaseMetadata(context: ExtensionContext, storagePath:
 
   const offlineCache = path.join(storagePath, 'latestApprovedRelease.cache.json');
 
-  async function readCachedReleaseData(): Promise<IRelease | null> {
+  async function readCachedReleaseData(): Promise<IRelease[] | null> {
```

Comment:
This breaks on existing installations. Maybe the simplest solution is to use a name different from  `latestApprovedRelease.cache.json` going forward.

Reply:
yeah, a different name will be more correct: `approvedReleases.cache.json`? (And `readCacheReleasesData` for function name?)